src/test/scala/dpla/ingestion3/wiki/WikiMapperTest.scala

```diff
@@ -1,15 +1,14 @@
 package dpla.ingestion3.wiki
 
-import dpla.ingestion3.data.EnrichedRecordFixture
-import dpla.ingestion3.model.{URI, _}
+import dpla.ingestion3.model._
 import org.scalatest.{FlatSpec, Matchers}
 
 /**
   * Tests for the ingestion3 FileIO utility
   */
-class WikiMapperTest extends FlatSpec with Matchers {
+class MapperTest extends WikiMapper
```

Comment:
Curious why the nested classes. Missed this until now. Also, that comment seems to not belong here!

Reply:
hmm  -- I thought I moved that nested class out in the last commit. Should be outside the test class now.